challenges/linked-list/linked-list.js

```diff
@@ -0,0 +1,42 @@
+class CurrentNode {
+  constructor(num){
+    this.num = num;
+  }
+}
+class LinkList {
+  constructor() {
+    this.head = null;
+    this.length = 0;
+  }
+
+
+  insert(num) {
+    const newNode = new CurrentNode(num);
+    newNode.next = this.head;
+    this.head = newNode;
+    this.length++;
+  }
+
+  includes(num) {
+    let currentNode = this.head;
+    for(let i = 0; i < this.length; i++){
+      if(currentNode.num === num){
+        return true;
+      } else {
```

Comment:
I think you want this else statement outside of the for loop - a return breaks a loop, which you want in the case of true but as you have it set up right now, it will evaluate the first thing, return true or return false, and then end the loop.